prisma-fmt/tests/code_actions/scenarios/one_to_one_referenced_side_misses_unique_single_field/diagnostics.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "range": {
+      "start": {
+        "line": 8,
```

Comment:
Ok I get there is a new diagnostics.json now, but I don't understand the difference with `result.json`